### PR TITLE
#14: updated home page

### PR DESCRIPTION
**Pull request by katherinetucker**
updated ui for home page

### CODE REVIEW

**Comment by ryanturner** [comment]
Hey @katherinetucker, will you please resolve the merge conflicts?

**Comment by katherinetucker** [comment]
@ryanturner conflict resolved

File: app/main/templates/home.html [hunk 1]
```diff
@@ -94,5 +119,24 @@ <h2 class="section-title">How do I submit an application?</h2>
       </div>
       <label >****NOTE: Incomplete Applications will NOT be considered and will be returned****</label>
     </div>
+    <br>
+    <center>
+    <div id="carousel">
```
Comment (ryanturner) [inline app/main/templates/home.html]: Dont use <br> or <center>; do the markup purely in CSS for your #carousel

Use top-padding and "margin: 0 auto;" to recreate the same thing.

**Comment by katherinetucker** [comment]
@ryanturner made changes that you requested

**Pull request merged**